devices/Ssd13xx/Ssd13xx.cs

```diff
@@ -42,13 +44,42 @@ public abstract class Ssd13xx : IDisposable
         /// </summary>
         public IFont Font { get; set; }
 
+        private GpioController _gpioController;        
+        private int _resetPin;
+        private bool _shouldDispose;
+        private bool _disposed = false;
+
         /// <summary>
         /// Constructs instance of Ssd13xx
         /// </summary>
         /// <param name="i2cDevice">I2C device used to communicate with the device</param>
+        /// <param name="resetPin">Reset pin (some displays might be wired to share the microcontroller's
+        /// reset pin).</param>
         /// <param name="resolution">Screen resolution to use for device init.</param>
-        public Ssd13xx(I2cDevice i2cDevice, DisplayResolution resolution = DisplayResolution.OLED128x64)
+        /// <param name="gpio">Gpio Controller.</param>
+        /// <param name="shouldDispose">True to dispose the GpioController.</param>
+
+        public Ssd13xx(
+            I2cDevice i2cDevice,
+            DisplayResolution resolution = DisplayResolution.None,
```

Comment:
for compatibility, would be better to use the previous selected resolution. I know you've adjusted the code but still